include/ginkgo/core/base/math.hpp

```diff
@@ -75,11 +75,13 @@ struct remove_complex_impl<std::complex<T>> {
 
 
 template <typename T>
-struct is_complex_impl : public std::integral_constant<bool, false> {};
+struct is_complex_impl : public std::integral_constant<bool, false> {
```

Comment:
Has this been run through `clang-format` ?

Reply:
Unfortunately, this is because of `clang-format`. I could not change it back for some reason.